package.json

```diff
@@ -33,7 +33,7 @@
   },
   "homepage": "https://github.com/open-feature/js-sdk#readme",
   "engines": {
-    "node": ">=16"
+    "node": ">=18"
```

Comment:
This arguably makes this a breaking change. What do you think about leaving it at 16 but taking the officially stance that we only support actively maintained versions of node?

Reply:
Mh thats a good question.
The problem is that we would have to wait for removing node 16 until the next release of a major would happen then. I guess this could be a pretty long time. On the other hand just breaking the SDK for this now also does not feel good.

Technically we could maybe separate provider Node version from SDK version to avoid this I guess.
In this case people using EOL Node 16 could still use the SDK but might not be able to use every provider.
This could be better than having a breaking change in the SDK.
On the other hand there will be a moment we will have to remove Node 16 but I am really not sure when this would be a good moment.

Reply:
So we could maybe change the Node version for contribs but keep this, what do you think @beeme1mr?
If we go for this I would still wonder when we should remove Node 16 from here.

Reply:
I'm mainly concerned about the sdk. Providers can specify their own minimum node version.

Reply:
So it might be an option to close this for now but merge https://github.com/open-feature/js-sdk-contrib/pull/797.
Still at some point we would then have to remove Node 16 support as it is EOL.

Reply:
What if we generally do it like OTEL here @beeme1mr? This way we would not have to have a new major if we remove a node version that is not supported anymore.

https://github.com/open-telemetry/opentelemetry-js?tab=readme-ov-file#supported-runtimes

Reply:
I don't think we should consider it breaking if we remove EOL runtime versions. We can put that somewhere in the spec or something.

Reply:
> I don't think we should consider it breaking if we remove EOL runtime versions. We can put that somewhere in the spec or something.
> 
> 
> 
> Removing actively supported versions is a different matter, and I think there's a stronger argument for considering that breaking, but it doesn't apply here.

Yes I think this is fine.
@toddbaert should we add a section like the OTEL docs I linked?